### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token '<' (Error) using version 0.17.22

### DISCUSSION THREAD

**Discussion by Ravinder74599** [GrapesJS/grapesjs#4551] [Q&A]
<img width="546" alt="Screenshot 2022-09-04 at 12 03 54 AM" src="https://user-images.githubusercontent.com/69594566/188283826-7ab25b1e-9330-441e-97d2-b92ab6bc0e62.png">
Only Happening on editor page.
Other pages are loading properly.
also the code working smoothly on local.
using Nginx server

**Comment by artf** [comment]
Without a reproducible demo looks like an issue with your application bundle.
